securedrop/i18n_tool.py

```diff
@@ -89,21 +89,28 @@ def translate_messages(self, args: argparse.Namespace) -> None:
                 *sources,
             )
 
-            sed("-i", "-e", '/^"POT-Creation-Date/d', messages_file)
+            msg_file_content = messages_file.read_text()
+            updated_content = _remove_from_content_line_with_text(
+                text='"POT-Creation-Date:', content=msg_file_content
+            )
+            messages_file.write_text(updated_content)
 
-            if self.file_is_modified(messages_file) and len(os.listdir(args.translations_dir)) > 1:
+            if (
+                self.file_is_modified(str(messages_file))
```

Comment:
Any reason you didn't modify file_is_modified to take Path objects?

Reply:
I didn't want to add more changes to this PR, but I can modify it if needed.

Reply:
That's a fine reason, just wanted to make sure you had considered it :)